packages/editor/src/components/collab-sidebar/index.js

```diff
@@ -0,0 +1,184 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	comment as commentIcon,
+	commentAuthorAvatar as userIcon,
+	Icon,
+	check as resolvedIcon,
+} from '@wordpress/icons';
+import { useSelect } from '@wordpress/data';
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	__experimentalText as Text,
+} from '@wordpress/components';
+import { dateI18n, format, getSettings } from '@wordpress/date';
+
+/**
+ * Internal dependencies
+ */
+import PluginSidebar from '../plugin-sidebar';
+import { store as editorStore } from '../../store';
+
+const isBlockCommentExperimentEnabled =
+	window?.__experimentalEnableBlockComment;
+
+/**
+ * Renders the Collab sidebar.
+ */
+export default function CollabSidebar() {
+	const { threads } = useSelect( ( select ) => {
+		const post = select( editorStore ).getCurrentPost();
+
+		return {
+			threads: post?.meta?.collab ? JSON.parse( post.meta.collab ) : [],
```

Comment:
Could this create problem if there's a large enough number of comments on a post? Could it be lazy loaded in some way to work around this problem? Thinking in terms of google docs, you have to scroll both on the page and on the comments dropdown to see all the comments.

Reply:
We have implemented a feature where the sidebar displays if any comment is open. However, we have now changed the flow so that the sidebar will be visible even if there are no collaborative activities.

Reply:
We have conducted testing and believe that a large number of comments will not have a significant impact. Once the flow is finalized, we can definitely implement lazy loading.